ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/PropertiesList.java

```diff
@@ -127,13 +126,12 @@ public class PropertiesList extends KsqlEntity {
       DEFAULT_DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG,
       DEFAULT_TIMESTAMP_EXTRACTOR_CLASS_CONFIG,
       FAIL_ON_DESERIALIZATION_ERROR_CONFIG,
-      KSQL_STRING_CASE_CONFIG_TOGGLE,
-      KSQL_NESTED_ERROR_HANDLING_CONFIG,
-      KSQL_QUERY_ERROR_MAX_QUEUE_SIZE,
-      KSQL_QUERY_RETRY_BACKOFF_INITIAL_MS,
-      KSQL_QUERY_RETRY_BACKOFF_MAX_MS,
       KSQL_QUERY_PULL_MAX_ALLOWED_OFFSET_LAG_CONFIG,
-      KSQL_QUERY_PULL_TABLE_SCAN_ENABLED,
```

Comment:
The only real change is to remove this config -- everything else is just moving the ksql configs that are out of alphabetical order to maintain readability

Reply:
how is this class used? these are still "per query" configurations in that you can set them per pull query. I see in the description you explicitly call out that they're not "persistent query" configurations, which is correct